Storage/rest/basic.h

```diff
@@ -123,7 +123,8 @@ struct Basic {
       input.entry.InitializeOwnKey();
       if (!Exists(input.field[current::storage::sfinae::GetKey(input.entry)])) {
         input.field.Add(input.entry);
-        return Response(current::ToString(current::storage::sfinae::GetKey(input.entry)), HTTPResponseCode.Created);
+        return Response(current::ToString(current::storage::sfinae::GetKey(input.entry)),
```

Comment:
P.S.
> Nobody is going to use it in the nearest future :)
I remember one of you telling someone to get the identifier right from the response body, it wasn't even a JSON.